### PR TITLE
Fix issue#50-Create a Description section of About Us

### LINKED ISSUE #50: Create a Description section of About Us
Add to the about us section the description of Flourish!

DoD:
- The Who We are Section uses the proper Header 1 styling
- The offset paragraph section contains the blurb following DoD
- The header and body are properly offset (say ~10% offset on the body)
- The body paragraph contains the proper font sizing

Body Text:
Flourish! is an annually held conference intendedc to introduce both CS and non-CS, technical and non-technical majors to the world of open source through speakers, demos, workshops, sponsors, job opportunities, and other events! We show off hardware, software, business, entrepreneurship, finance, and more aspects of open source. We want to make this project bigger and bigger year over year and bring in new students. Our previous conferences focused on a small number of students in a technical software direction, but we are looking to change that!

Make sure you are looking at the figma design for dimensions/colors/etc. Feel free to reach out to the project lead if you have questions!

### PR DESCRIPTION
**Pull request by Jeongjae-0124**
Created a Description section of About us resolves #50

### CODE REVIEW

**Comment by ChristianBingman** [comment]
No properly attached issue, and description/title is inadequate.

File: README.md [hunk 1]
```diff
@@ -97,4 +97,4 @@ Ron Pham - [@Bacoburn](https://github.com/Bacoburn)
 
 Adrian Knight - [@Ajknight121](https://github.com/Ajknight121)
 
-Ares Stavropoulos - [@Ares1605](https://github.com/Ares1605)
\ No newline at end of file
+Ares Stavropoulos - [@Ares1605](https://github.com/Ares1605)
```
Comment (ChristianBingman) [inline README.md:100]: I am going to ignore this for now, but double check that your IDE isn't messing with things.

File: src/components/Description/index.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+export default function Description(){
+    return (<div className={"place-items-center m-auto"}>
+                <div className={"container m-auto px-[5rem]"}>
+                    <div className={"mx-[5rem] min-w-full"}>
+                        <h1>Description</h1>
+                    </div>
+                    <div className={"ml-[10rem] min-w-fit"}>
+                        <body>Flourish! is an annually held conference intendedc to introduce both CS and non-CS, technical and non-technical majors to the world of open source through speakers, demos, workshops, sponsors, job opportunities, and other events! We show off hardware, software, business, entrepreneurship, finance, and more aspects of open source. We want to make this project bigger and bigger year over year and bring in new students. Our previous conferences focused on a small number of students in a technical software direction, but we are looking to change that!</body>
```
Comment (ChristianBingman) [inline src/components/Description/index.jsx:9]: Didn't realize the who we are section used body tags, but that is incorrect and can cause problems down the road. Could you please change this to something else? (p would be best)

File: src/components/Description/index.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+
+export default function Description(){
+    return (<div className={"place-items-center m-auto"}>
```
Comment (ChristianBingman) [inline src/components/Description/index.jsx:3]: This is redundant since the parent also has this.

File: src/pages/about.js [hunk 4]
```diff
@@ -1,8 +1,9 @@
 import './about.css'
 import SpeakersSection from '../components/SpeakersSection';
 import ExecutiveLeaders from '../components/ExecutiveLeaders';
-
+import Descpription from '../components/Description';
```
Comment (ChristianBingman) [inline src/pages/about.js:4]: Description is misspelled

File: src/pages/about.js [hunk 5]
```diff
@@ -12,6 +13,7 @@ export default function About(){
                         <body>The Flourish! Council is a small group of ~10 students who are interested in open source technologies and want to bring this knowledge to other students! We are all UIC students ranging from 1st year to graduate with varying levels of experience. All with a common goal in of teaching about open source! Our SIG WebDev created this site as our first official project aimed at increasing interest through a fun and informative website! See below for more information about the executive project leaders and our amazing speakers!</body>
                     </div>
                 </div>
+                <Descpription/>
```
Comment (ChristianBingman) [inline src/pages/about.js:16]: Misspelling again.

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Few small changes, thank you for putting this in a component! Also, your git tree is pretty mangled, you should start a new branch and reset to main after this is merged.

File: README.md (hunk 1 above)
Comment (Jeongjae-0124) [reply README.md:100]: I don't know why this is being changed, because I haven't modified anything from the ReadMe file

**Review by ChristianBingman — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**